pkg/api/graph.go

```diff
@@ -167,6 +174,14 @@ func StableImageTagLink(name, tag string) StepLink {
 	}
 }
 
+func Comparer() cmp.Option {
```

Comment:
I can see an argument for the functions in the env utils package to live in here, so we don't need to export this, WDYT

Reply:
I think I prefer the current state. Exporting a `Comparer` is not a big deal, and the current layout makes sense to me.